app/exec.py

```diff
@@ -133,6 +133,8 @@ def autostart_conditionally(self) -> bool:
         return False
 
     def toggle_scroll_mode(self) -> bool:
+        if not self._terminal:
+            return self._scroll_mode
         if self._scroll_mode:
             self._terminal.exit_copy_mode()
             self._scroll_mode = False
```

Comment:
The if statement below checking if _terminal exists is redundant now

Reply:
nice catch thanks!